cardano-testnet/test/cardano-testnet-test/cardano-testnet-test.hs

```diff
@@ -63,6 +64,7 @@ tests = do
                 , ignoreOnWindows "Gov Action Timeout" Gov.hprop_check_gov_action_timeout
                 , ignoreOnWindows "Treasury Withdrawal" Gov.hprop_ledger_events_treasury_withdrawal
                 , ignoreOnWindows "PParam change fails for SPO" Gov.hprop_check_pparam_fails_spo
+                , ignoreOnWindows "Treasury Donation" Gov.hprop_ledger_events_treasury_donation
```

Comment:
small nitpick, can you move it after `Treasury Withdrawal` 😄

Reply:
I moved it before actually :+1: since it made more sense lexicographically-order wise.